### ISSUE TITLE
bug: Import list callback is called three times

### ISSUE THREAD

**Issue by JaythFactor** [ffxiv-teamcraft/ffxiv-teamcraft#2839]
**Describe the bug**
Whenever I use the import list feature with a callback query string, the endpoint provided as the callback receives three identical POST requests.

**To Reproduce**
1. Open an import link with a callback endpoint provided. ex: `https://ffxivteamcraft.com/import/MjA1NDUsbnVsbCwzOzE3OTYyLDMyMzA4LDE7MjAyNDcsbnVsbCwx?callback=https://example.com/list-created`
2. Complete list import.
3. Open the browser console then notice that there are three instances of the request failure error. If you use an endpoint that accepts the requests, notice in your server logs that the endpoint is called three times.

**Expected behavior**
I would expect the provided endpoint to be called only once when the list import is completed.

**Screenshots**
Three request attempts made using the example url above:
![Screenshot 2024-06-20 111326](https://github.com/ffxiv-teamcraft/ffxiv-teamcraft/assets/61891879/d30a19a1-13b4-4e48-8e6d-4cb8ecd223f8)

**Desktop:**
 - OS: Windows
 - Browser: Firefox
 - Version: N/A(Website)

**Comment by Supamiu** [comment]
This is not a bug, it's due to the request having a `retry(3)` operator, meaning that if the request fails, it'll try until it reaches 3 errors in total, that's the expected behavior.

**Comment by Supamiu** [comment]
Here is the implementation for this behavior: https://github.com/ffxiv-teamcraft/ffxiv-teamcraft/blob/staging/apps/client/src/app/core/interceptor/error-interceptor.ts#L23

**Comment by JaythFactor** [comment]
Oh, I see! I just wasn't returning a response so it thought there was an error and retried. Easy fix to return success. It would be nice to have this mentioned in the wiki page documentation.

Thanks for the help!
